equinix/helpers_device.go

```diff
@@ -68,64 +69,46 @@ type NetworkInfo struct {
 	PrivateIPv4    string
 }
 
-func getNetworkInfoMetalGo(ips []metalv1.IPAssignment) NetworkInfo {
+func getNetworkInfo(ips []metalv1.IPAssignment) NetworkInfo {
```

Comment:
I cleaned up the unnecessary MetalGo suffixes from helper methods, but it does make some of these changes harder to review.  I can back out the name change if that's preferred, but I'm going to instead try to annotate the current changes to guide review.